test-apps/display-test-app/android/imodeljs-test-app/app/copyAssets.gradle

```diff
@@ -0,0 +1,50 @@
+//file:noinspection GroovyGStringKey
+import java.nio.file.Files
+import java.nio.file.Paths
+import java.nio.file.StandardCopyOption
+
+static def copyDirectory(srcDir, destDir) {
+    Files.walk(srcDir).forEach(srcPath -> {
+        def destinationPath = Paths.get(destDir.toString(), srcDir.relativize(srcPath).toString())
+        // skip copying directories that already exist
+        if (!Files.exists(destinationPath) || !Files.isDirectory(destinationPath))
+            Files.copy(srcPath, destinationPath, StandardCopyOption.REPLACE_EXISTING)
+    })
+}
+
+static def copyFile(srcFile, destDir) {
+    def destFile = Paths.get(destDir.toString(), srcFile.getFileName().toString())
+    Files.copy(srcFile, destFile, StandardCopyOption.REPLACE_EXISTING)
+}
+
+task copyAssets {
+    doLast {
+        def dtaDir = "${rootDir}/../.."
+        println "dtaDir ${new File(dtaDir).canonicalPath}"
+        def assetsDir = "${rootDir}/app/src/main/assets"
+        println "assetsDir ${new File(assetsDir).canonicalPath}"
+
+        Map foldersToCopy = [
+                "${dtaDir}/lib/mobile/main.js": "${assetsDir}/backend",
```

Comment:
I'm guessing that this came from the iTwin Mobile samples, but I'm uncomfortable with "foldersToCopy" having an entry with a file on one side and a folder on the other. If it needs to be a file, then the filename should be listed on both sides, and the variable should be renamed to something like itemsToCopy. If "/main.js" shouldn't be on the left, then it should be removed.

Reply:
I've renamed the variable to itemsToCopy. The copyFile function takes a source file and a destination folder so it doesn't need to be listed on both sides.